eng/pipelines/templates/jobs/archetype-sdk-client.yml

```
@@ -164,12 +168,25 @@ jobs:
           testRunTitle: "$(OSName) $(TestTargetFramework)"
           testResultsFormat: "VSTest"
           mergeTestResults: true
+      - template: /eng/pipelines/templates/steps/select-feed.yml
+      - task: UniversalPackages@0
+        condition: and(succeededOrFailed(), eq(variables['CollectCoverage'], 'true'), eq(variables['coverage.collected'], 'true'))
+        displayName: Get code coverage history
+        continueOnError: true
+        inputs:
+          command: download
+          downloadDirectory: $(Coverage.HistoryDirectory)
+          feedsToUse: internal
+          vstsFeed: $DevOpsFeedID
```

```suggestion
          vstsFeed: $(DevOpsFeedID)
```
This is a devops variable so I think you need the parens.

---

Oops